### PR TITLE
Modify scopes in oidc-sample-app.properties

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Add email scope and address scope to the oidc-sample-app.properties file to obtain email and address user attributes.

### CODE REVIEW

File: docs/integrating_with_existing_webapp.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The structure of the oidc-sample-app we are configuring would be as follows:
         errorPage=
         logoutURL=logout
         callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
-        scope=openid,internal_application_mgt_view
+        scope=openid,address,email,internal_application_mgt_view
```
Comment (mefarazath) [inline docs/integrating_with_existing_webapp.md]: Btw i was wondering whether we need to request `internal_application_mgt_view` at all?

**Review by emswbandara — APPROVED** [review]

File: docs/integrating_with_existing_webapp.md (hunk 1 above)
Comment (emswbandara) [reply docs/integrating_with_existing_webapp.md]: @ChamathNS any idea why internal_application_mgt_view scope is needed?

**Review by JKAUSHALYA — APPROVED** [review]

File: docs/integrating_with_existing_webapp.md (hunk 1 above)
Comment (chamathns) [reply docs/integrating_with_existing_webapp.md]: We ported the same params from our existing OIDC web apps (pickup dispatch, manager, etc).
This scope was added along with that IINM. I also think we don't need this one.
+1 for removing

Comment (Thisara-Welmilla) [reply docs/integrating_with_existing_webapp.md]: Address with commit ef0556e58d4730c07bf43454d7ba242614ccd2a1

**Review by mefarazath — APPROVED** [review]

**Pull request merged**